components/brave_wallet_ui/page/wallet_page.html

```diff
@@ -3,12 +3,23 @@
   <head>
   <base href="/">
   <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width">
+  <if expr="is_ios">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+  </if>
+  <else>
```

Comment:
Is `<else>` valid preprocessor syntax? 🤔 I think when you use it it has to be used with `<then>` instead?

Reply:
example: https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/browser_resources.grd;l=110